src/cqparts/display/cqparts_server.py

```diff
@@ -47,31 +49,59 @@ def display_callback(self, component):
         if ENVVAR_SERVER not in os.environ:
             raise KeyError("environment variable '%s' not set" % ENVVAR_SERVER)
 
-        # Verify Parameter(s)
-        # check that it is a component
-        from .. import Component
-        if not isinstance(component, Component):
-            raise TypeError("given component must be a %r, not a %r" % (
-                Component, type(component)
-            ))
+        # get the server from the environment
+        server_url = os.environ[ENVVAR_SERVER]
+        # check that the server is running
+        try:
+            requests.get(server_url + '/status')
```

Comment:
This `try` block only needs this line.
If the later requests raise a `ConnectionError`, but this line doesn't... then I think throwing the exception higher is warranted.

Reply:
I should really look in the responses too, but yes.